ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/ParcelableMemoryCache.java

```diff
@@ -107,15 +192,31 @@ public Token storeBitmap(@Nullable final Bitmap documentBitmap) {
         if (documentBitmap != null) {
             mBitmapCache.put(token, documentBitmap);
         }
+        LOG.debug("Store bitmap for token {}", token);
+        logCacheSizes();
+        return token;
+    }
+
+    @NonNull
+    public Token storeBitmap(@Nullable final Bitmap documentBitmap, @NonNull final String tag) {
+        final Token token = Token.next(tag);
+        if (documentBitmap != null) {
+            mBitmapCache.put(token, documentBitmap);
+        }
+        LOG.debug("Store bitmap for token {} with tag {}", token, tag);
+        logCacheSizes();
         return token;
     }
 
     public void removeBitmap(@NonNull final Token token) {
         mBitmapCache.remove(token);
+        LOG.debug("Remove bitmap for token {}", token);
+        logCacheSizes();
```

Comment:
I can imagine that the logs helped during development, but I'm not sure if every method should be logged here and if it is worth to pollute the logcat so much.